src/containers/containerCollection/containerCollection.jsx

```diff
@@ -6,13 +6,13 @@ import PropTypes from 'prop-types'
 import { Grid, Typography, Card, CardHeader, CardContent, CardActions, Button } from '@material-ui/core'
 import { withStyles } from '@material-ui/core/styles'
 import { withSnackbar } from 'notistack';
-import {SortableContainer, SortableElement} from 'react-sortable-hoc';
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
 
 import Container from 'components/container/container'
 import EditContainer from 'components/editContainer/editContainer'
-import { addContainer, deleteContainer } from 'actions/container/container'
```

Comment:
how come we don't need deleteContainer anymore?

Reply:
Because now it's imported and dispatched from container, instead of being passed thru props from containerCollection.